src/concept/type/Type.jl

```diff
@@ -23,6 +23,21 @@ function instantiate(t::Proto._Type)
     end
 end
 
+"""
+    ==(type1::AbstractType, type2::AbstractType)
+secures the equality of two types by comparing the two labels and the
+same type.
+"""
+function ==(type1::AbstractType, type2::AbstractType)
```

Comment:
You probably mean to extend from Base here?

```suggestion
function Base.:(==)(type1::AbstractType, type2::AbstractType)
```

Reply:
Yes, you are right. Changed it to your suggestions.